### PR TITLE
Contact Section (mobile)

### PR DESCRIPTION
**Pull request by nottherealalanturing**
Added forms, background images and styles to the contact section on the mobile site.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -278,5 +278,37 @@ <h6>Skills</h6>
...
+            name="email"
+            id="email"
+            placeholder="example@email.com"
+          />
+          <textarea
+            name="message"
+            id="message"
+            cols="20"
+            rows="10"
+            placeholder="Wrtie your message here"
```
Comment (ShoiraTa) [inline index.html]: - [x] Based on requirements for this milestone, all input fields should have the required attribute to activate the validation on submit. Please make sure to add required attribute for all three inputs.

File: index.html [hunk 2]
```diff
@@ -278,5 +278,37 @@ <h6>Skills</h6>
...
+          that you need built or a project that needs coding. I’d love to help
+          with it
+        </p>
+        <form action="https://formspree.io/f/xlezbjdv" method="POST">
+          <input
+            type="text"
+            name="fullname"
+            id="fullname"
+            placeholder="Your Name"
+          />
```
Comment (ShoiraTa) [inline index.html]: - [x] Name input (can be first name and last name field depending on the template used) should have `maxlength="30"` attribute. Please add `maxlength="30"` attribute 😉

File: index.html [hunk 3]
```diff
@@ -278,5 +278,37 @@ <h6>Skills</h6>
...
+            type="email"
+            name="email"
+            id="email"
+            placeholder="example@email.com"
+          />
+          <textarea
+            name="message"
+            id="message"
+            cols="20"
+            rows="10"
```
Comment (ShoiraTa) [inline index.html:309]: - [x] Please add `maxlength="500"` attribute in textarea.

**Review by ShoiraTa — CHANGES_REQUESTED** [review]
Hi team,

:thumbsup: This is good to make it this far, your project is really nice, congratulations on the good work you have done so far

There are some issues that you still need to work on to go to the next project but you are almost there!



 ## Required Changes  :recycle: 

_Check the comments under the review._ 

***
Cheers and Happy coding! :clap: :clap: :clap:

Feel free to leave any questions or comments in the PR thread if something is not 100% clear. 

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

* * *
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by ReshyResh — APPROVED** [review]
# Hi @nottherealalanturing , 

### Good job addressing the changes requested by the previous reviewer!
#### Your project is complete!  There is nothing else to say other than... it's time to merge it!
Congratulations! 🎉 


Cheers and Happy coding!👏👏👏

----

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)_

**Pull request merged**